src/modules/position_estimator_inav/position_estimator_inav_main.cpp

```diff
@@ -410,7 +410,7 @@ int position_estimator_inav_thread_main(int argc, char *argv[])
 
 	/* wait for initial baro value */
 	bool wait_baro = true;
-	TerrainEstimator *terrain_estimator = new TerrainEstimator();
```

Comment:
You think so? Look at TerrainEstimator, it contains only some floats and timestamps, <100B.